GO_Sync/src/main/java/edu/usf/cutr/go_sync/io/GTFSReadIn.java

```diff
@@ -129,26 +76,31 @@ public String readAgency(String agency_fName)
     }
 
     public List<Stop> readBusStop(String fName, String agencyName, String routes_fName, String trips_fName, String stop_times_fName){
-        Hashtable<String, HashSet<Route>> stopIDs = new Hashtable<String, HashSet<Route>>();
+        long tStart = System.currentTimeMillis();
         Hashtable<String, HashSet<Route>> id = matchRouteToStop(routes_fName, trips_fName, stop_times_fName);
-        stopIDs.putAll(id);
+        Hashtable<String, HashSet<Route>> stopIDs = new Hashtable<String, HashSet<Route>>(id);
 
         String thisLine;
         String [] elements;
         int stopIdKey=-1, stopNameKey=-1, stopLatKey=-1, stopLonKey=-1;
         try {
             BufferedReader br = new BufferedReader(new InputStreamReader(new FileInputStream(fName),"UTF-8"));
-            boolean isFirstLine = true;
-            Hashtable<String,Integer> keysIndex = new Hashtable<String,Integer>();
-            while ((thisLine = br.readLine()) != null) {
-                if (isFirstLine) {
-                    isFirstLine = false;
-                    if (thisLine.startsWith(UTF8_BOM)) {
-                        thisLine = thisLine.substring(1);
-                    }
-                    OperatorInfo.setGtfsFields(thisLine);
-                    thisLine = thisLine.replace("\"", "");
-                    String[] keys = thisLine.split(",");
+            boolean isFirstLine = false;//true;
+            Hashtable<String,Integer> keysIndex = new Hashtable<String,Integer> ();
```

Comment:
Generally Java recommends using `HashMap` instead of `Hashtable`. Was there a specific reason for using `Hashtable`?